src/placement/ring_map.c

```diff
@@ -123,8 +123,8 @@ ring_comp_shuff_cmp(struct pool_component *comp_a,
 		    struct pool_component *comp_b,
 		    unsigned int seed, unsigned int prime)
 {
-	uint64_t	key_a = comp_a->co_id;
-	uint64_t	key_b = comp_b->co_id;
+	uint64_t        key_a = comp_a->co_id;
+	uint64_t        key_b = comp_b->co_id;
```

Comment:
why do you change all tabs to spaces? I really prefer to use tabs and only touch the code which is supposed to be changed